pkg/apis/policy/v1alpha1/types_certificaterequestpolicy.go

```diff
@@ -172,118 +174,123 @@ type CertificateRequestPolicyAllowedX509Subject struct {
 	// +optional
 	PostalCodes *CertificateRequestPolicyAllowedStringSlice `json:"postalCodes,omitempty"`
 
-	// SerialNumber defines the X.509 Subject Serial Number that may be requested
-	// for.
+	// SerialNumber defines the X.509 Subject Serial Number that may be
+	// requested for.
 	// +optional
 	SerialNumber *CertificateRequestPolicyAllowedString `json:"serialNumber,omitempty"`
 }
 
-// CertificateRequestPolicyAllowedStringSlice represents an allowed string
-// slice value paired with whether the field is a required value on the
-// request.
+// CertificateRequestPolicyAllowedStringSlice represents allowed string values
+// paired with whether the field is a required value on the request.
+// An empty Values paired with Required `true` is an "always-false" condition.
+// always denies.
```

Comment:
This line is misplaced or missing words or the punctuation on the line before it is wrong

Reply:
Removed it now.